lib/galaxy/datatypes/binary.py

```diff
@@ -4302,3 +4302,24 @@ def display_peek(self, dataset: DatasetProtocol) -> str:
             return dataset.peek
         except Exception:
             return "Binary Numpy npz file (%s)" % (nice_size(dataset.get_size()))
+
+
+@build_sniff_from_prefix
+class FITS(Binary):
+    """
+    FITS (Flexible Image Transport System) file data format, widely used in astronomy
+    Represents sky images (in celestial coordinates) and tables
+    https://fits.gsfc.nasa.gov/fits_primer.html
+    """
+
+    file_ext = "fits"
+
+    def sniff_prefix(self, file_prefix: FilePrefix):
```

Comment:
```suggestion
    def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
```
Can you add this return type, please?